shop/src/main/java/ro/msg/learning/shop/dto/ProductIdAndQuantity.java

```diff
@@ -0,0 +1,17 @@
+package ro.msg.learning.shop.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.UUID;
+
+@Data
+@AllArgsConstructor
+@Builder
+@NoArgsConstructor
+public class ProductIdAndQuantity {
```

Comment:
is this a dto?  the product id and quantity is also available in the stock dto. why use this one as well?

Reply:
because my input is a list of products (product ID and quantity) contained in the order and i decided to make a dto for this and not using the stockDto which includes the locationId(in my case is null)